workspace/src/github.com/darkcrux/go-ceph-rest-api/handler_pool.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+  "fmt"
+  "net/http"
+  "github.com/gorilla/mux"
+  "encoding/json"
+)
+
+const (
+  // actions
+  create = "create"
+  // get = "get"
+  // delete = "delete"
+
+  // methods
+  // get = "GET"
+  // post = "POST"
+  put = "PUT"
+  // delete = "DELETE"
+)
+
+type PoolAPI struct {
+  Res http.ResponseWriter
+  Req *http.Request
+  Params map[string]string
+}
+
+type Response struct {
+  Status string      `json:"status"`
+  Output interface{} `json:"output"`
+}
+
+func poolHandler(res http.ResponseWriter, req *http.Request) {
+  res.WriteHeader(200)
```

Comment:
We should use proper HTTP response codes. It would save a lot of trouble. ;)

HTTP 200 = Response Success (eg. List Pool Success)
HTTP 201 = Created (eg. Pool created, etc)

HTTP 400 = Bad Request (wrong parameters, etc)
HTTP 401 = Unauthorized

HTTP 503: Service Unavailable
... etc.

More info here: http://en.wikipedia.org/wiki/List_of_HTTP_status_codes

We should avoid HTTP 500 though.